addresses/goerli.json

```diff
@@ -132,11 +132,11 @@
 					"timeStretch": 26
 				},
 				"ytPool": {
-					"address": "0x4c3f56cc1ad2bc61293739c7f0550575c9daf313",
+					"address": "0x4c3f56cC1ad2BC61293739C7f0550575c9DAf313",
 					"poolId": "0x4c3f56cc1ad2bc61293739c7f0550575c9daf313000200000000000000000078",
```

Comment:
what about the poolid?

Reply:
Discussed on discord, Pool ids are formatted to all lower case by balancer.